tests/ansible/regression/issue_655__wait_for_connection_error.yml

```diff
@@ -0,0 +1,85 @@
+# https://github.com/dw/mitogen/issues/655
+# Spins up a Centos8 container and runs the wait_for_connection test inside of it
+# Doing it this way because the shutdown command causes issues in our tests
+#   since things are ran on localhost; Azure DevOps loses connection and fails
+# TODO: do we want to install docker a different way to be able to do this for other tests too
+---
+# this should only run on our Mac hosts
```

Comment:
What prevents this from running on non-mac hosts? I don't see any safety checks to bail if the host is not a mac, instead it will just go up in smoke when the dmg can't be unpacked, or the /Volumes directory doesn't exist.

Reply:
The `target` host only runs on our Mac setup because Mac is the only test that runs `localhost_ansible` tests (see `azure-pipelines.yml`). Technically someone could run it themselves on a non-mac and it might not work but the tests are really set up for running through CI mostly so I didn't worry about that. I left a note so that if we end up running `localhost_ansible` on another host then we'd know to refactor this file a bit! It's not super clear though; if you'd like I could edit the comment to reflect something different!

Reply:
Nah. Comment is enough. :)

Reply:
Sounds good!